packages/pg-meta/src/query/Query.utils.ts

```diff
@@ -166,6 +166,22 @@ function applyFilters(query: string, filters: Filter[]) {
   if (filters.length === 0) return query
   query += ` where ${filters
     .map((filter) => {
+      // Handle composite values
+      if (Array.isArray(filter.column)) {
+        switch (filter.operator) {
+          case 'is':
```

Comment:
Is there a source for the operators that aren't supported with tuples? If so might be worth documenting with a comment there or switch more to a whitelist support where we accept the filters that we know to work and reject the rest.

Alternatively, could it make sense to just let Postgres reject bad filters?

Reply:
good point, let's make it a whitelist